install.sh

```diff
@@ -2,34 +2,34 @@
 
 # Source setting.sh
 source /vagrant/settings.sh
+
+# Fetching LAN_IP and network address
+if [[ ${STANDALONE,,} != yes ]]; then
+   LAN_IP=$(ip addr | grep "inet\b" | cut -d" " -f6 | grep 192 |  cut -d/ -f1)
```

Comment:
You assumed LAN_IP will always class C 192.168.yyy.zzz. One can set 172.xxx.yyy.zzz

try
```ip addr show <interface> | grep -Po 'inet \K[\d.]+'
```

Reply:
I am not sure if the interface name would be same everytime. Hardcoding interface does not seem like a good idea.

Reply:
Yes, interface is the same every time, and it is `LAN_IF`.
The code proposed by Akash is correct

Reply:
Using this is also a good idea I think. Bcoz bridged interface will be always on 3 position.
`ip addr | grep -Po -m3 'inet \K[\d.]+' | tail -n1`

Reply:
No, it is not a good idea, because it never guarantied that the order of the interfaces will be the same. Do as Akash and me suggested: `ip addr show $LAN_IF`

Reply:
Yes. you are right.